stories/area/2_with_time.tsx

```diff
@@ -19,14 +19,15 @@
 
 import React from 'react';
 
-import { AreaSeries, Axis, Chart, Position, ScaleType, timeFormatter } from '../../src';
+import { AreaSeries, Axis, Chart, Placement, Position, ScaleType, Settings, timeFormatter } from '../../src';
 import { KIBANA_METRICS } from '../../src/utils/data_samples/test_dataset_kibana';
 import { SB_SOURCE_PANEL } from '../utils/storybook';
 
 const dateFormatter = timeFormatter('HH:mm');
 
 export const Example = () => (
   <Chart className="story-chart">
+    <Settings tooltip={{ stickTo: Position.Top, placement: Placement.RightStart, offset: 5 }} rotation={0} />
```

Comment:
Can we add knobs here?

```suggestion
    <Settings
      tooltip={{
        stickTo: getPositionKnob('stickTo'),
        placement: getPlacementKnob(),
        offset: number('offset', 5),
      }}
      rotation={0}
    />
```

Reply:
done in https://github.com/elastic/elastic-charts/pull/1122/commits/5067c16b18b7fed03ed99d07b52fb964723a24ab